tests/e2e/framework/kubernetes.go

```diff
@@ -487,7 +492,16 @@ func (k *KubeInfo) Teardown() error {
 		// clean up using helm
 		err := util.HelmDelete(istioHelmChartName)
 		if err != nil {
-			return nil
+			// If fail don't return so other cleanup activities can complete
```

Comment:
good improvement - although a little concerned this might destabilize testiing.

Reply:
It was destablizing by returning nil.  Basically it would return without any of the other cleanup.  Especially the Istio namespace cleanup.